automated-comments/csharp/two-fer/use_single_formatted_string_not_multiple.md

```diff
@@ -0,0 +1 @@
+There are two `return` statements returning very similar `string`'s. Could you try to structure your code such that this duplication is eliminated? This will involve having just one `return` statement, returning a single `string`.
```

Comment:
@mikedamay I've tried to rephrase a bit. What do you think of this version (where I try to be a bit more explicit about the return statements)?

Reply:
That sounds good.  I am sure if I had seen the target code, your original would have made sense.  I had something involving the ternary operator in mind (which I have seen quite often) when I made my comment.

Reply:
```
`string`'s
``` doesn't need an apostrophe

Reply:
Rather than

> Could you try to structure your code such that this duplication is eliminated? 

make it less personal with:

> How could this code be structured so that this duplication is eliminated?